src/containers/Questions/questions.css

```
@@ -0,0 +1,162 @@
+.question__type {
+    border-radius: 6px;
+    min-height: 200px;
+    text-align: center;
+    padding-top: 0px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center
+}
+
+.question__error-field {
+    font-size: 10px;
```

the font is too small here 
```suggestion
    font-size: 14px;
```